manta-accounting/src/wallet/signer.rs

```diff
@@ -1111,6 +1175,35 @@ where
         Ok(senders)
     }
 
+    /// Builds two virtual [`Sender`]s for `pre_sender`.
+    #[inline]
+    fn virtual_senders(
+        &mut self,
+        parameters: &Parameters<C>,
+        asset_id: &C::AssetId,
+        pre_sender: PreSender<C>,
+    ) -> Result<[Sender<C>; PrivateTransferShape::SENDERS], SignError<C>>
+    where
+        UtxoAccumulatorModel<C>: Clone,
```

Comment:
Instead of cloning, we should just have a function like

```rust
pub fn empty(model: &Self::Model) -> Self;
```

on the `Accumulator` trait that does the right thing.